apps/prairielearn/src/sprocs/assessment_groups_copy.sql

```diff
@@ -32,14 +32,22 @@ BEGIN
     RETURNING id INTO temp_old_group_config_id;
 
     -- soft delete the old groups
-    UPDATE groups g
-    SET deleted_at = NOW()
-    WHERE g.group_config_id = temp_old_group_config_id;
-
+    WITH deleted_groups AS (
+      UPDATE groups g
+      SET deleted_at = NOW()
+      WHERE g.group_config_id = temp_old_group_config_id
+    )
     INSERT INTO group_logs 
-        (authn_user_id, group_id, action)
-    VALUES 
-        (assessment_groups_copy.authn_user_id, temp_old_group_config_id, 'delete');
```

Comment:
This feels just completely broken to me; we're inserting a `group_configs.id` value as `group_id`?

Given that this has already produced completely invalid data, I have half a mind to drop the `group_logs` table entirely (or rename it to `group_logs_old`) and start over from scratch. Not in this PR, of course.

Given _that_, maybe I don't bother making this change at all? 🤷

Reply:
Tangent: as discussed on #6256, copying is just totally broken. We could consider entirely removing the copying functionality for now until we can rewrite it "correctly" and avoid this whole discussion?

Reply:
Actually, _because_ this code is so broken, it has never produced any logs 🙃

Reply:
#8910 will remove copying entirely, meaning we don't have to worry about this at all. Yay!

Reply:
After the merge in 7037811, we no longer have to deal with this at all.